# Request
string model_name
string model_link
string ee_name
string ee_link
---
# Reply
bool success
string message
